examples/counter-json1/server.js

```diff
@@ -0,0 +1,41 @@
+var http = require('http');
+var express = require('express');
+var ShareDB = require('sharedb');
+var WebSocket = require('ws');
+var WebSocketJSONStream = require('@teamwork/websocket-json-stream');
+var json1 = require('ot-json1')
+
+ShareDB.types.register(json1.type);
+var backend = new ShareDB();
+createDoc(startServer);
+
+// Create initial document then fire callback
+function createDoc(callback) {
+  var connection = backend.connect();
+  var doc = connection.get('examples', 'counter');
+  doc.fetch(function(err) {
+    if (err) throw err;
+    if (doc.type === null) {
+      doc.create({numClicks: 0}, 'json1', callback);
```

Comment:
It is slightly safer to specify the type URI `json1.type.uri` instead of the type name `'json1'`, just in case of a forked type with the same name.

That's what happens if no type is provided and `create()` uses the default json0 type:
https://github.com/share/sharedb/blob/55188897cd219ff2b59c16da9bf5d6a6403b8e40/lib/client/doc.js#L853-L855

Reply:
Done.